src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final String DEFAULT_DELIMITER = "[,:]";
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int CUSTOM_DELIMITER = 1;
+    private static final int SPLIT_TARGET = 2;
+
+    private StringAddCalculator() {}
+
+    public static int splitAndSum(String text) {
+        if (isEmpty(text)) {
+            return 0;
+        }
+
+        if (hasSingle(text)) {
+            return parseInt(text);
+        }
+
+        return sum(split(text));
+    }
+
+    private static boolean isEmpty(String text) {
+        if (text == null) {
+            return true;
+        }
+        if (text.isEmpty()) {
+            return true;
+        }
+        if (text.isBlank()) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return text == null || text.isBlank();
```
조건이 간단하기 때문에 inline으로 처리할 수 있습니다.

`isBlank`가 empty도 포함하고 있기 때문에 `isBlank` 하나로도 커버가 가능합니다.